src/scrape_up/hacker_news/articles.py

```diff
@@ -0,0 +1,32 @@
+import requests
+from bs4 import BeautifulSoup
+import json
+
+class Article:
+
+    def __init__(self):
+        self.help = "This scrapes articles"
+
+    def __scrap_page(self):
+        data = requests.get(
+            "https://news.ycombinator.com/"
+        )
+        data = BeautifulSoup(data.text, "html.parser")
+        return data
```

Comment:
Return the result or the response as shown in the image below
![image](https://github.com/Clueless-Community/scrape-up/assets/93156825/3f6fcb61-0dec-4fd2-9532-befdafff5130)